qtrubyshared is a real library and sholud be SONAMED. patch backported from trunk with the change that
we use 1d instead of 2.
--- a/ruby/qtruby/src/CMakeLists.txt
+++ b/ruby/qtruby/src/CMakeLists.txt
@@ -32,6 +32,7 @@
 # target_link_libraries(qtrubyinternal ${QT_QTCORE_LIBRARY} ${RUBY_LIBRARY} ${QT_QTNETWORK_LIBRARY})
 
 set_target_properties(qtruby4 PROPERTIES PREFIX "")
+set_target_properties(qtruby4shared PROPERTIES VERSION 1d.0.0 SOVERSION 1d)
 
 install(TARGETS qtruby4shared LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX} )
 install(TARGETS qtruby4 DESTINATION ${CUSTOM_RUBY_SITE_ARCH_DIR})
